src/protocols/device_path.rs

```diff
@@ -64,3 +64,19 @@ pub struct HardDriveMedia {
     pub partition_format: u8,
     pub signature_type: u8,
 }
+
+pub struct Media {
+    pub header: Protocol,
+}
+
+impl Media {
+    pub const SUBTYPE_HARD_DRIVE: u8 = 0x01;
+    pub const SUBTYPE_CD_ROM: u8 = 0x02;
+    pub const SUBTYPE_VENDOR: u8 = 0x03;
+    pub const SUBTYPE_FILE_PATH: u8 = 0x04;
+    pub const SUBTYPE_MEDIA_PROTOCOL: u8 = 0x05;
+    pub const SUBTYPE_PIWG_FIRMWARE_FILE: u8 = 0x06;
+    pub const SUBTYPE_PIWG_FIRMWARE_VOLUME: u8 = 0x07;
+    pub const SUBTYPE_RELATIVE_OFFSET_RANGE: u8 = 0x08;
+    pub const SUBTYPE_RAMDISK: u8 = 0x09;
```

Comment:
Opposite case here, EDK2 uses `RAM_DISK`. There are some other definitions that use `RAMDISK`, but the majority uses `RAM_DISK`. I think we should follow this style.

Reply:
Amended as requested in latest push.